cmd/server/main.go

```diff
@@ -23,21 +24,20 @@ import (
 	"github.com/Azure/kubernetes-kms/pkg/version"
 
 	"google.golang.org/grpc"
-	logsapi "k8s.io/component-base/logs/api/v1"
-	json "k8s.io/component-base/logs/json"
-	"k8s.io/klog/v2"
 	kmsv1 "k8s.io/kms/apis/v1beta1"
 	kmsv2 "k8s.io/kms/apis/v2"
+	"monis.app/mlog"
 )
 
 var (
-	listenAddr    = flag.String("listen-addr", "unix:///opt/azurekms.socket", "gRPC listen address")
-	keyvaultName  = flag.String("keyvault-name", "", "Azure Key Vault name")
-	keyName       = flag.String("key-name", "", "Azure Key Vault KMS key name")
-	keyVersion    = flag.String("key-version", "", "Azure Key Vault KMS key version")
-	managedHSM    = flag.Bool("managed-hsm", false, "Azure Key Vault Managed HSM. Refer to https://docs.microsoft.com/en-us/azure/key-vault/managed-hsm/overview for more details.")
-	logFormatJSON = flag.Bool("log-format-json", false, "set log formatter to json")
+	listenAddr   = flag.String("listen-addr", "unix:///opt/azurekms.socket", "gRPC listen address")
+	keyvaultName = flag.String("keyvault-name", "", "Azure Key Vault name")
+	keyName      = flag.String("key-name", "", "Azure Key Vault KMS key name")
+	keyVersion   = flag.String("key-version", "", "Azure Key Vault KMS key version")
+	managedHSM   = flag.Bool("managed-hsm", false, "Azure Key Vault Managed HSM. Refer to https://docs.microsoft.com/en-us/azure/key-vault/managed-hsm/overview for more details.")
+	logLevel     = flag.String("log-level", "", "In order of increasing verbosity: unset (empty string), info, debug, trace and all.")
 	// TODO remove this flag in future release.
+	_              = flag.Bool("log-format-json", false, "set log formatter to json")
```

Comment:
It seems that the only available log level for servers in mlog is JSON. Therefore, my suggestion would be to disregard the flag and consider deprecating it in the future. wdyt?